swagger-templates/src/main/java/com/okta/swagger/codegen/AbstractOktaJavaClientCodegen.java

```diff
@@ -789,6 +789,14 @@ public void postProcessModelProperty(CodegenModel model, CodegenProperty propert
                 model.imports.add(type);
             }
 
+            if (model.name.equals("CAPTCHAInstance")) {
+                model.imports.remove("CAPTCHAInstanceLink");
+            }
+
+            if (model.name.equals("HrefObject")) {
+                model.imports.remove("HrefObjectHints");
+            }
```

Comment:
what does this do specifically?

Reply:
This removes the unnecessary imports for these objects which have already become a part of the `com.okta.sdk.resource.c.ap.tc.ha` package using the code associated with the previous comment.

Reply:
This code is removed and hence this is no longer applicable.